signing_service/signing_service/signing_service.py

```diff
@@ -491,6 +496,20 @@ def _parse_arguments() -> argparse.Namespace:
         help=f'Environment which will be set in Raven client config `environment` parameter.',
     )
 
+    subparsers = parser.add_subparsers(help='sub-command help')
+
+    # create the parser for the console notifier
+    parser_for_local_notifier = subparsers.add_parser('console-notifier', help='default console notifier')
+    parser_for_local_notifier.add_argument('--from-email-address', default=None)
```

Comment:
This and below arguments are unnecessary for `console-notifier` - please remove them. `ConsoleNotifier` doesn't need all that stuff.

Reply:
@dybi There are some troubles with that part. Because base on what am I suppose to chose which `Notifier` to use? `subparsers` is not a variable I can check. 

- Right now I can check if `from_email_address` and ` from_email_password` are not `None`, so that I can use `EmailNotifier`, else console.

- Or we could change it by adding additional default argument that specify what did we chose. Something like `--notifier`

Reply:
yes, we should have an argument to specify which notifier to choose. And depending on its value use corresponding subparser.

Reply:
As we discuss, I've deleted whole `console-notifier` subparser because it's our default and has no additional arguments.